k9mail/src/main/java/com/fsck/k9/message/PgpMessageBuilder.java

```diff
@@ -33,6 +33,7 @@
 
     public static final int REQUEST_SIGN_INTERACTION = 1;
     public static final int REQUEST_ENCRYPT_INTERACTION = 2;
+    public static final int REQUEST_INLINE_INTERACTION = 3;
```

Comment:
INLINE refers to inline sign or inline verification? The German Wikipedia page suggests it can do both: https://de.wikipedia.org/wiki/PGP/INLINE